packages/suite-data/src/guide/constants.ts

```diff
@@ -0,0 +1,5 @@
+import { resolve, join } from 'path';
+
+export const GB_SOURCE = 'git@github.com:trezor/trezor-suite-guide.git';
```

Comment:
Without reading docs in the last commit, I thought that GB means British English. (probably I am just stupid)

Do we need to be that specific in the code? that we use the abbreviation of GitBook?

Reply:
Good point. Renamed to `GITBOOK_SOURCE` and `GITBOOK_REVISION` in 1b01e2d9fe4eb15ae4da7c53a02a92a02f894a4c.